documentation/guide-lombok.asciidoc

```diff
@@ -0,0 +1,74 @@
+:toc: macro
+toc::[]
+
+= Lombok 
+
+Lombok is a library that works with an annotation processor and will generate getter and setter for your classes. Lombok can save you some time and reduce the amount of boilerplate code in your project.
```

Comment:
Generating getter and setter is just one feature of Lombok. So better formulate this a bit different